ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -817,6 +817,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy
+- as: gcp-ipi-proxy-workload-identity-arm-f28
+  cron: 59 12 14 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      E2E_RUN_TAGS: '@arm64 and @gcp-ipi and @proxy and @network-ovnkubernetes and
+        not @fips'
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity
+- as: gcp-ipi-proxy-workload-identity-tp-amd-f28-destructive
+  cron: 55 0 22 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @proxy and @network-ovnkubernetes and
+        not @fips'
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
we do not need the TP flag, right?

Reply:
The env on `ipi-conf-gcp-oidc-creds-provision` step is only needed when running some TP feature, and we have to enable it, the workload-identity already GAed, so we should never need to set it.

Reply:
Removed the TP tag, thanks!